tests/unit/services/session-test.js

```diff
@@ -84,6 +84,32 @@ describe('SessionService', function() {
           expect(setupAnalyticsStub).to.have.been.calledWith(user);
         });
       });
+
+      it('sets up Launch Darkly with _setupLaunchDarkly', function() {
+        const _setupLaunchDarklyStub = sinon.stub();
+        subject.set('_setupLaunchDarkly', _setupLaunchDarklyStub);
+
+        const promise = subject.loadCurrentUser();
+        return promise.then(() => {
+          expect(_setupLaunchDarklyStub).to.have.been.calledWith(user);
+        });
+      });
+
+      it("calls Launch Darkly's identify via _setupLaunchDarkly", function() {
+        const identifyStub = sinon.stub();
+        subject.set('launchDarkly.identify', identifyStub);
+
+        const promise = subject._setupLaunchDarkly(user);
```

Comment:
```suggestion
        const promise = subject.loadCurrentUser(user);
```

Re: your question about two tests: 
I think the first test is not necessary...we don't care that it calls `_setupLaunchDarkly` specifically. What we care about is that when `loadCurrentUser` is called, it calls the right thing on the launch darkly service. So changing line 102 to call `loadCurrentUser` and checking that the identify method got called, seems sufficient to me.

Reply:
@cadeParade That test would fail.  The `loadCurrentUser` returns before the `_setupLaunchDarkly` async function is fully resolved.  Do you know how to nicely make it wait?